src/pages/home/report/ReportActionContextMenu/ReportActionContextMenu.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {Pressable, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../../styles/styles';
+import getReportActionContextMenuStyles from '../../../../styles/getReportActionContextMenuStyles';
+import getButtonState from '../../../../libs/getButtonState';
+import ContextActions from './ContextActions';
+import variables from '../../../../styles/variables';
+import Tooltip from '../../../../components/Tooltip';
+
+const propTypes = {
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportID: PropTypes.number.isRequired,
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportActionID: PropTypes.number.isRequired,
+    isMini: PropTypes.bool,
+    shouldShow: PropTypes.bool,
+};
+
+const defaultProps = {
+    isMini: false,
+    shouldShow: false,
+};
+
+const ReportActionContextMenu = (props) => {
+    const {wrapperStyle, getButtonStyle} = getReportActionContextMenuStyles(props.isMini);
```

Comment:
Not sure about returning a function and a style from this. Something about this feels like it could be organized better. I'll try to think of something more useful to share. 😂